### PR TITLE
umbrella create Tweaks

### PR DESCRIPTION
**Pull request by pgherveou**

### CODE REVIEW

File: umbrella/Cargo.toml [hunk 1]
```diff
@@ -538,19 +539,8 @@ with-tracing = [
...
-	"frame-election-provider-support",
-	"frame-executive",
-	"frame-metadata-hash-extension",
 	"frame-support",
-	"frame-support-procedural",
-	"frame-support-procedural-tools-derive",
 	"frame-system",
-	"frame-system-benchmarking",
-	"frame-system-rpc-runtime-api",
-	"frame-try-runtime",
```
Comment (pgherveou) [inline umbrella/Cargo.toml:553]: @kianenigma @ggwpe a few tweaks:  these crates don't need to be in the runtime features as well, as they are already included in `polkadot-sdk-frame`, they were also causing some annyoning warning when compiling

Comment (gui1117) [reply umbrella/Cargo.toml:553]: features names are not global. If the `frame-benchmarking` feature is not activated in this Cargo.toml then even if `polkadot-sdk-frame` has a feature with the same name, then the feature won't be activated. This means when compiled with feature `runtime`, the umbrella will not reexport `frame-benchmarking`. `frame-benchmarking` will only be available from `polkadot-sdk-frame`. Is it what we want?

File: substrate/frame/support/procedural/Cargo.toml [hunk 2]
```diff
@@ -69,6 +69,11 @@ experimental = []
 # pallets in a runtime grows. Does increase the compile time!
 tuples-96 = []
 tuples-128 = []
+try-runtime = [
+	"frame-support/try-runtime",
+	"frame-system/try-runtime",
+	"sp-runtime/try-runtime"
+]
```
[the remark this answers is not in the document]
Comment (pgherveou) [reply substrate/frame/support/procedural/Cargo.toml:76]: that's added by zepter, automatically, I explicity added try-runtime feature in the cargo as it complains in some situation with
```
warning: unexpected `cfg` condition value: `try-runtime`
   --> substrate/frame/support/procedural/src/lib.rs:323:31
```

File: umbrella/Cargo.toml (hunk 1 above)
Comment (pgherveou) [reply umbrella/Cargo.toml:553]: > frame-benchmarking. frame-benchmarking will only be available from polkadot-sdk-frame. Is it what we want?

Yeah I think that is fine to access all frames thing through polkadot-sdk-frame

File: substrate/frame/support/procedural/Cargo.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (gui1117) [reply substrate/frame/support/procedural/Cargo.toml:76]: in the past it has been avoided see this comment: https://github.com/paritytech/polkadot-sdk/pull/5099#discussion_r1687126137

But in this exact case, it doesn't harm. But if it then get used for different usecase, then I don't know.

Note that we could fix it like this: https://github.com/paritytech/polkadot-sdk/compare/pg/umbrella-tweak...gui1117:gui-fix-try-runtime-usage-in-frame-support?expand=1

**Pull request closed without merging**